pytrickle/server.py

```diff
@@ -443,6 +450,9 @@ async def _handle_update_params(self, request: web.Request) -> web.Response:
                     "message": "No active stream to update"
                 }, status=400)
             
+            # Clear input queues before updating to avoid processing stale frames
+            await self.current_client.clear_input_queues()
```

Comment:
As we discussed, I think this should be a setting in the update params to allow for turning this off.  I don't know that every update to params we should drop all frames in the input queue.